scripts/ci-docker-images/builder-x86_64-unknown-linux-musl/Dockerfile

```diff
@@ -336,7 +339,7 @@ ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER=gnu-cc
 
 # Set up linker and enable LTO plugin in rustc if necessary.
 ARG LTO
-ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_RUSTFLAGS="${LTO:+-Clinker-plugin-lto }-Clink-arg=${LTO:+-f$LTO}"
+ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_RUSTFLAGS="${LTO:+-Clinker-plugin-lto -Clink-arg=-O3}-Clink-arg=${LTO:+-f$LTO}"
```

Comment:
just for my info, can you explain why you added this flag?

Reply:
It passes `-O3` flag to the linker which enables all link-time optimizations, even those that improve performance at expense of increasing binary size. The default optimization level is not so aggressive. It is consistent with `opt-level = 3` for `rustc` which is set by Cargo by default for release builds.

Reply:
👍 awesome, thanks for the explanation. 😄